django_project/sql_scripts/20_core_utils_schema.sql

```diff
@@ -211,14 +203,15 @@ BEGIN
 
         ELSEIF v_result_type = 'Text'
             THEN
-                v_text_value = v_new_attributes -> v_key;
+                -- for whatever reason text values must be extracted as text (oprerator ->>)
```

Comment:
v_text_value is of type text, the function return type is also of type text
`-> returns json, ->> returns text`

so it makes sense to use your data as text instead json